README.md

```diff
@@ -95,3 +106,38 @@ method for the pull request and delete the branch after merge.
           title: Test pull request
           body: Test pull request body
 ```
+
+### Creating pull requests in a remote repository
+
+Create a `new-branch` branch with the changes to the remote repository. Note,
```

Comment:
I suggest using the word "different" instead of "remote" in this sentence to avoid confusion. The term "remote" has a specific meaning in git that does not apply to the action being described, as the process here involves creating a branch and a pull request on a separate repository, rather than creating a branch locally and making a pull request against a remote repository.

Reply:
Thanks, I've reworded this part